lib/pos_epoch.go

```diff
@@ -88,6 +88,10 @@ func (epochEntry *EpochEntry) GetEncoderType() EncoderType {
 	return EncoderTypeEpochEntry
 }
 
+func (epochEntry *EpochEntry) isBlockHeightInEpoch(blockHeight uint64) bool {
```

Comment:
Since the receiver function is on the EpochEntry, it's good naming convention to frame the function as "does this EpochEntry contain the input block height"
```suggestion
func (epochEntry *EpochEntry) ContainsBlockHeight(blockHeight uint64) bool {
```